.travis.yml

```diff
@@ -1,15 +1,22 @@
-addons:
-  apt:
-    packages:
-      - expect-dev
+cache: pip
 
 language: python
 
 python:
   - "2.7"
-  - "3.4"
```

Comment:
Why removing 3.4 and 3.6? I use 3.6 personally

Reply:
@lasley because Odoo officially supports 2.7 and 3.5.

Reply:
Is there harm in leaving 3.6? 

They only officially support the outdated wkhtmltopdf v0.12.1, but I can attest that 0.12.4 works perfectly and does not leave my system insecure.

Reply:
@lasley I don't know... For the lib itself, it seems safe.

Reply:
Odoo's commitment is >=3.5 even if they test only 3.5. So keep 3.6.

Reply:
We'll also need to remove the 3.4 classifier from [setup.py](https://github.com/LasLabs/oca-decorators/pull/1/files#diff-2eeaed663bd0d25b7e608891384b7298L29)

If the commitment is >=3.5, should we think about adding `3.7-dev` to Travis and `3.7` to the setup identifiers?

Reply:
@lasley 

> leave my system insecure like the earlier version

Unrelated but can you share any pointer about those security issues?

Reply:
@lasley The 3.4 classifier is removed from setup.py

Reply:
If you're using static binaries (or Windows), OpenSSL and Libpng come bundled. Assuming you're using 0.12.1.2, the OpenSSL is [1.0.1j](https://www.cvedetails.com/vulnerability-list/vendor_id-217/product_id-383/version_id-178264/Openssl-Openssl-1.0.1j.html)

Reply:
If you're default *nix installation you're fine FYI. It's dynamic by default AFAIK